harness/determined/core/_train.py

```diff
@@ -16,6 +19,19 @@ class EarlyExitReason(enum.Enum):
     USER_REQUESTED_STOP = "EXITED_REASON_USER_REQUESTED_STOP"
 
 
+class TensorboardSyncMode(enum.Enum):
```

Comment:
Does the word "Sync" belong in the name?  It actually governs both sync an metric writing, right?  Maybe just `TensorboardMode`?

Reply:
I changed it to `TensorboardMode` and now I am like, wait a minute, writing _is_ part of sync in some sense, isn't it? idk, let's go with `TensorboardMode`.